### ISSUE TITLE
Update the documentation.

### ISSUE THREAD

**Issue by kujtimiihoxha** [jadjoubran/laravel5-angular-material-starter#238]
I would suggest to update the documentation to add explanation on how to override bower dependencies `main` I noticed that a lot of bower packages do not add the necessary  files in their `main` so when bower task runs it will not automatically include these files. An example would be bootstrap, if we want bootstrap to include the necessary  files we would write. 

```
{
  "name": "my app",
  "version": "0.0.0",
  "dependencies": {
    "bootstrap": "*"
  },
  "overrides": {
    "bootstrap": {
      "main": [
        "dist/css/bootstrap.css",
        "dist/js/bootstrap.js"
      ]
    }
  }
}

```

I think this should be in the documentation because this is a very common issue.

**Comment by jadjoubran** [comment]
Hi @kujtimiihoxha 
Thank you very much for this, indeed it is important
it also reminds me of #205 
I will update both 3.0 and 3.1 documentations to mention this :+1:

**Comment by kujtimiihoxha** [comment]
Also this works if the package has dependencies that are not mentioned in the `bower.json` ex. some package is dependent on  `jquery` but it is not in the packages bower.json we could write 

```
{
  "name": "my app",
  "version": "0.0.0",
  "dependencies": {
    "example-package": "*"
  },
  "overrides": {
    "example-package": {
      "dependencies":  {
    "jquery": "*"
     }
    }
  }
}
```

this will allow that the jquery library is loaded before the example-package.

**Comment by kujtimiihoxha** [comment]
@jadjoubran Yes this is the solution to #205. Also I needed to copy the fonts to the public folder.

**Comment by jadjoubran** [comment]
Fixed 3.1 and 3.0 docs :+1:
